includes/lib/usage-tracking/class-usage-tracking-base.php

```diff
@@ -423,163 +419,69 @@ private function get_plugin_name( $basename ) {
 
 	/**
 	 * Hide the opt-in for enabling usage tracking.
+	 *
+	 * @deprecated 3.1.0 - Opt-in moved to Setup Wizard
 	 **/
 	protected function hide_tracking_opt_in() {
-		update_option( $this->hide_tracking_opt_in_option_name, true );
+		_deprecated_function( __METHOD__, '3.1.0', 'Sensei_Onboarding::skip_setup_wizard' );
 	}
 
 	/**
 	 * Determine whether the opt-in for enabling usage tracking is hidden.
 	 *
-	 * @return bool true if the opt-in is hidden, false otherwise.
+	 * @deprecated 3.1.0 - Opt-in moved to Setup Wizard
 	 **/
 	protected function is_opt_in_hidden() {
-		return (bool) get_option( $this->hide_tracking_opt_in_option_name );
```

Comment:
Could we also remove the `$hide_tracking_opt_in_option_name` attribute and where it gives it a value in the constructor (`$this->hide_tracking_opt_in_option_name = $this->get_prefix() . '_usage_tracking_opt_in_hide';`)?

Reply:
Got rid of that too.